scan/templates/blocks/detail.html

```diff
@@ -7,10 +7,13 @@
 
 {% block content %}
 
-  <h5 class="p-2">Block #{{ blk.id }}</h5>
+
```

Comment:
you are omitting the blocks id? I mean we always use the height and not the id....so what

Reply:
good catch was in the wrong template - weill revert

Reply:
Did a new commit for this file